EmpleoDotNet/Views/Home/Index.cshtml

```diff
@@ -18,9 +18,20 @@
                     {
                         <a href="@Url.Action("Detail", "JobOpportunity", new {id = jobOpportunity.Id})" class="featured" target="_blank">
                             <div class="row">
-                                <div class="col-md-1 hidden-sm hidden-xs">
-                                    <img src="@jobOpportunity.CompanyLogoUrl" width="60" height="60" alt="" class="img-responsive" onclick="onJobOpportunityDetailClick(this);" data-url="@Url.Action("Detail", "JobOpportunity", new { id = jobOpportunity.Id})" />
+                                <div class="col-md-1 hidden-sm hidden-xs text-center">
+                                    @if (!string.IsNullOrEmpty(jobOpportunity.CompanyLogoUrl))
+                                    {
+                                        <img src="@jobOpportunity.CompanyLogoUrl" width="60" height="60" alt="" class="img-responsive" onclick="onJobOpportunityDetailClick(this);" data-url="@Url.Action("Detail", "JobOpportunity", new { id = jobOpportunity.Id })" />
+                                    }
+                                    else
+                                    {
+                                        var splited = jobOpportunity.CompanyName.Split(' ');
+                                        var lg = splited.Count() > 1 ? $"{splited[0].Substring(0, 1)}{splited[1].Substring(0, 1)}" : splited[0].Substring(0, 1);
+
+                                        <h5 style="color:#0f858d; font-size:30px">@lg</h5>
+                                    }
```

Comment:
Mis notas:
* Creo que seria mas apropiado mover la logica del substring a una propiedad aparte en un viewmodel para no tenerla incrustada en el html.
* El if statement completo se puede mover a una clase helper de razor ([mas informacion aqui](http://www.asp.net/web-pages/overview/ui-layouts-and-themes/creating-and-using-a-helper-in-an-aspnet-web-pages-site)) para que pueda reutilizarse en la pagina de detalle de ese empleo.
* Cambiar el inline styling por una clase css con un nombre descriptivo.